modules/packages/msiexec.bat

```diff
@@ -0,0 +1,170 @@
+@echo off
+setlocal ENABLEDELAYEDEXPANSION
+
+rem Use real applications or testing stub for testing
+
+set MSIEXEC=msiexec
+set WMIC=wmic
+set CSCRIPT=cscript
+
+if not "%CFENGINE_MSIEXEC_TEST%"=="" (
+  set MSIEXEC=%CFENGINE_MSIEXEC_TEST% msiexec
+  set WMIC=%CFENGINE_MSIEXEC_TEST% wmic
+  set CSCRIPT=%CFENGINE_MSIEXEC_TEST% cscript
+)
+
+rem choose a function to execute
+
+if "%1"=="supports-api-version" <nul set /p=1

```

Comment:
I think this should just do `echo 1`. I've tested that and it works fine. Any reasons for this `set /p` magic?

Reply:
agreed. This magic prints single byte to output, without CRLF symbols.